lib/MusicBrainz/Server/Entity/Relationship.pm

```diff
@@ -224,20 +244,19 @@ sub _interpolate {
         }
     }
     my %extra_attrs = %attrs;
-    my $type_is_orderable = $self->link->type->orderable_direction > 0;
 
     # Ordered relationships in a series should all share the same link phrase,
     # even if their attributes differ, so that they remain grouped together
     # in the relationships display.
-    %attrs = () if $type_is_orderable;
+    %attrs = () if $for_grouping;
```

Comment:
I would move this comment to the above `_build_grouping_phrase` subroutine and make a more general comment about dropping unnecessary attributes for grouping here.

Reply:
Should be better now :)